pkg/apis/kubeone/v1alpha1/types.go

```diff
@@ -215,6 +215,7 @@ type MachineControllerConfig struct {
 
 // Features controls what features will be enabled on the cluster
 type Features struct {
+	PodNodeSelector   *PodNodeSelector   `json:"podNodeSelector"`
```

Comment:
While there is no technical obstacle to add this new feature to v1alpha1, I think we should avoid doing this, as v1alpha1 should be "conserved" and not changed anymore.

Reply:
I've thought about this as well, but I think we should keep both APIs in sync until we don't officially announce the deprecation with the 1.0 release. After that, only v1beta1 should be extended.